===================================================================
RCS file: /cvs/glibc/libc/misc/mktemp.c,v
retrieving revision 1.3
retrieving revision 1.4
diff -u -r1.3 -r1.4
--- libc/misc/mktemp.c	1999/07/06 15:26:06	1.3
+++ libc/misc/mktemp.c	2000/09/26 06:47:24	1.4
@@ -32,3 +32,5 @@
 
   return template;
 }
+
+link_warning (mktemp, "the use of `mktemp' is dangerous, better use `mkstemp'")
===================================================================
RCS file: /cvs/glibc/libc/stdio-common/tempnam.c,v
retrieving revision 1.8
retrieving revision 1.9
diff -u -r1.8 -r1.9
--- libc/stdio-common/tempnam.c	1999/07/06 15:26:08	1.8
+++ libc/stdio-common/tempnam.c	2000/09/26 06:51:40	1.9
@@ -39,3 +39,6 @@
 
   return __strdup (buf);
 }
+
+link_warning (tempnam,
+	      "the use of `tempnam' is dangerous, better use `mkstemp'")
===================================================================
RCS file: /cvs/glibc/libc/stdio-common/tmpnam.c,v
retrieving revision 1.9
retrieving revision 1.10
diff -u -r1.9 -r1.10
--- libc/stdio-common/tmpnam.c	1999/07/06 15:26:12	1.9
+++ libc/stdio-common/tmpnam.c	2000/09/26 06:52:46	1.10
@@ -45,3 +45,6 @@
 
   return s;
 }
+
+link_warning (tmpnam,
+	      "the use of `tmpnam' is dangerous, better use `mkstemp'")
===================================================================
RCS file: /cvs/glibc/libc/stdio-common/tmpnam_r.c,v
retrieving revision 1.6
retrieving revision 1.7
diff -u -r1.6 -r1.7
--- libc/stdio-common/tmpnam_r.c	1999/07/06 15:26:13	1.6
+++ libc/stdio-common/tmpnam_r.c	2000/09/26 06:53:27	1.7
@@ -33,3 +33,6 @@
 
   return s;
 }
+
+link_warning (tmpnam_r,
+	      "the use of `tmpnam_r' is dangerous, better use `mkstemp'")
